src/cets.erl

```diff
@@ -243,12 +245,22 @@ insert_many(Server, Records) when is_list(Records) ->
 %% extra messaging is required.
 -spec insert_new(server_ref(), tuple()) -> WasInserted :: boolean().
 insert_new(Server, Rec) when is_tuple(Rec) ->
-    Res = cets_call:send_leader_op(Server, {insert_new, Rec}),
+    Res = cets_call:send_leader_op(Server, {leader_op, {insert_new, Rec}}),
     handle_insert_new_result(Res).
 
 handle_insert_new_result(ok) -> true;
 handle_insert_new_result({error, rejected}) -> false.
 
+%% @doc Serialized version of `insert/2'.
+%%
+%% All `insert_serial' calls are sent to the leader node first.
+%%
+%% Similar to `insert_new/2', but overwrites the data silently on conflict.
+%% It could be used to update entries, which use not node-specific keys.
+-spec insert_serial(server_ref(), tuple()) -> ok.
+insert_serial(Server, Rec) when is_tuple(Rec) ->
+    ok = cets_call:send_leader_op(Server, {insert, Rec}).
```

Comment:
So we send the operation to the leader node, but we skip the `leader_op` check. Why?

Reply:
good catch. Generally, I wanted to avoid match for true | false, returned by insert_new. But I've also skipped is_leader check, added a test and a fix.

Reply:
also, apparently, insert also returns true (instead of ok). So, I don't need to match for ok in the return before replicating in handle_leader_op. Ah, prolog api :)